Dockerfile

```diff
@@ -6,6 +6,6 @@ COPY . .
 RUN pip3 install --user -r requirements.txt && pip3 cache purge
 ENV PYTHONPATH="/usr/local:$PYTHONPATH"
 
-CMD python vtkw-server.py --port 1234 --host 0.0.0.0
+CMD python src/opengeodeweb_viewer/rpc/schemas/vtkw-server.py --port 1234 --host 0.0.0.0
```

Comment:
why moving the server into rpc/schemas ? and not directly into opengeodeweb_viewer ?